### ISSUE TITLE
NPM peer dependency conflict with strapi

### ISSUE THREAD

**Issue by broksonic21** [mattmilburn/strapi-plugin-preview-button#116]
When I try updating from 1.1.3 to 2.1.1, I get a peer dependency conflict with npm with latest @strapi plugins
```
  "dependencies": {
    "@strapi/plugin-graphql": "4.14.5",
    "@strapi/plugin-i18n": "4.14.5",
    "@strapi/plugin-users-permissions": "4.14.5",
    "@strapi/provider-email-nodemailer": "4.14.5",
    "@strapi/provider-upload-cloudinary": "4.14.5",
    "@strapi/strapi": "4.14.5",
    "mysql": "2.18.1",
    "strapi-plugin-populate-deep": "^3.0.0",
    "strapi-plugin-preview-button": "2.1.1"
  },
```

```
npm ERR! Found: strapi-plugin-preview-button@1.1.3
npm ERR! node_modules/strapi-plugin-preview-button
npm ERR!   strapi-plugin-preview-button@"2.1.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! strapi-plugin-preview-button@"2.1.1" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react-intl@6.5.1
npm ERR! node_modules/react-intl
npm ERR!   peer react-intl@"^6.4.4" from strapi-plugin-preview-button@2.1.1
npm ERR!   node_modules/strapi-plugin-preview-button
npm ERR!     strapi-plugin-preview-button@"2.1.1" from the root project
```

**Comment by ParijatSaxena** [comment]
use this option: npm add strapi-plugin-preview-button@latest --legacy-peer-deps

**Comment by mattmilburn** [comment]
Hi @broksonic21 I have been unable to reproduce this issue. I can also confirm that Strapi and it's internal packages explicitly use `react-intl@6.4.1` not `6.5.1` like in your error. Do you have `react-intl` installed in your `package.json` or maybe another plugin installed that might be conflicting with this?

Although I do have to say... I'm surprised you're not getting a different error because this plugin expects `^react-intl@6.4.4` which is not exactly a match for `react-intl@6.4.1` 🤔  I may need to adjust that in the next release to use `^react-intl@6.4.1`.

**Comment by mattmilburn** [comment]
Hi again @broksonic21 This should be fixed now in the latest [version 2.2.0](https://github.com/mattmilburn/strapi-plugin-preview-button/releases/tag/v2.2.0) of this plugin 🎉  Let me know how it goes for you 👍🏻

**Comment by broksonic21** [comment]
Looks great @mattmilburn  - thanks!
